Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,31 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            //Start program
+            Console.WriteLine("Hello World! Welcome to the first C# 2020 Bootcamp Homework.");
+            PromptUser();
+            Console.WriteLine("Okay, Person #2:");
+            PromptUser();
+        }
+        static double GetBMI(double weight, int height) //calculates the BMI by receiving weight and height as parameters and dividing weight by height squared (kg/m^2)
+        {
+            double hSquared = (height / 100d); //convert cm to meters
+            hSquared *= hSquared; //square
+            return (weight / hSquared);
```

Comment:
()'s are redundant here

Reply:
True, but I think it improves user readability.